src/StdEnclosureView.m

```diff
@@ -81,6 +81,9 @@ -(void)setEnclosureFile:(NSString *)newFilename
     NSString * encodedname = [basename stringByReplacingPercentEscapesUsingEncoding:NSUTF8StringEncoding];
     NSString * ext = [encodedname pathExtension];
 
+	if (basename==nil)
```

Comment:
I think it is nicer to have `{ }` for the `if` statement, or keep it all one line so the logic is easier to understand. 
eg.
```
if (basename == nil) {
    return;
```
or
`if (basename == nil) return;`

Is this OK?